lib/lower/PFTBuilder.cpp

```diff
@@ -160,11 +160,22 @@ class PFTBuilder {
     parents.emplace_back(unit);
     return true;
   }
+  /// Make funclist to point to current parent function list if it exists.
+  void resetFuncList() {
```

Comment:
This feels more like a set than a reset to me. This is setting the backlink to the parent, right? Was it set someplace else prior to getting here?

Reply:
Yes, prior to this call, it was set to the funclist of the module/function that is being leaved. So `setFunctListToParentFuncs` may be the more accurate name here.